src/server/wtr-config.js

```diff
@@ -82,16 +82,22 @@ export class WTRConfig {
 						<style>
 
 							html {
+								--d2l-cursor-resize-inline-end: e-resize;
```

Comment:
Hopefully the resize ones are obvious.

The `--d2l-length-factor` is a little more general, and can be used to invert a dimension where CSS logical properties do not apply (for example, properties of `box-shadow`).